microsoft/testsuites/ltp/ltp.py

```diff
@@ -340,14 +344,29 @@ def _install(self) -> bool:
         self.node.tools[Mkdir].create_directory(top_src_dir, sudo=True)
         self.node.tools[Chmod].update_folder(top_src_dir, "a+rwX", sudo=True)
 
-        # clone ltp
-        git = self.node.tools[Git]
-        ltp_path = git.clone(
-            self.LTP_GIT_URL, cwd=PurePosixPath(top_src_dir), dir_name=top_src_dir
-        )
+        if self._tar_file_path:
+            self._log.debug(f"Use downloaded ltp tar: {self._tar_file_path}!")
+            self._remote_tar_path = self.get_tool_path(
+                use_global=True
+            ) / os.path.basename(self._tar_file_path)
+
+            mkdir = self.node.tools[Mkdir]
+            mkdir.create_directory(self._remote_tar_path.parent.as_posix())
+            self.node.shell.copy(PurePath(self._tar_file_path), self._remote_tar_path)
```

Comment:
It doesn't need to copy, just extract it to the target place.

Reply:
Do you mean direct extract from local env to remote node? That may cause lots of extracted files transfering which is very time consuming.

Reply:
ok, but use the tool called RemoteCopy